netmiko/dell/dell_powerconnect.py

```diff
@@ -27,7 +27,7 @@ def session_preparation(self):
     def set_base_prompt(self, pri_prompt_terminator='>', alt_prompt_terminator='#',
                         delay_factor=1):
         """Sets self.base_prompt: used as delimiter for stripping of trailing prompt in output."""
-        prompt = super(DellPowerConnectSSH, self).set_base_prompt(
+        prompt = super(CiscoSSHConnection, self).set_base_prompt(
```

Comment:
This should be `DellPowerConnectBase` i.e. you call super using the class name of the class you are currently in.